src/tests/ftest/util/ior_test_base.py

```diff
@@ -49,10 +49,14 @@ def setUp(self):
         self.subprocess = self.params.get("subprocess", '/run/ior/*', False)
         self.ior_timeout = self.params.get("ior_timeout", '/run/ior/*', None)
 
-    def create_pool(self):
-        """Create a TestPool object to use with ior."""
+    def create_pool(self, ranks=None):
+        """Create a TestPool object to use with ior.
+
+           Args:
+               ranks (str): Ranks on which to create the pool
+        """
         # Get the pool params and create a pool
-        self.add_pool(connect=False)
+        self.add_pool(connect=False, target_list=ranks)
```

Comment:
Will you instead call this directly in your test code to avoid modifying `ior_test_base`?
```
self.add_pool(connect=False, target_list=ranks)
```
We're trying to remove wrappers like this from test bases.
Benefits:
- test bases are simpler
- there is no risk of affecting tests that use common test bases

Reply:
Sure, done.

Reply:
Thanks :)